python/housinginsights/sources/base.py

```diff
@@ -145,7 +145,7 @@ def result_to_csv(self, fields, results, filepath):
             writer = csv.writer(f, delimiter=',')
             writer.writerow(fields)
             for result in results:
-                writer.writerow(result.data)
+                writer.writerow(result)
```

Comment:
The generic base.py had custom logic for DCHousing (knowing it had an internal field "data"). Now this just takes an array of data, so more generically usable.